src/components/UserData/DropdownList.tsx

```diff
@@ -0,0 +1,41 @@
+import classNames from 'classnames';
+import React from 'react';
+import { User } from '../../types/User';
+import { Loader } from '../Loader';
+
+type Props<T> = {
+  list: T[] | null
+  chosenElem: T | null
+  handler: (elem: (T | null)) => () => void
+};
+
+const DropdownList:React.FC<Props<User>> = React.memo((
+  {
+    list,
+    chosenElem,
+    handler,
+  },
+) => {
+  return (
+    <div className="dropdown-content">
+      {list
+        && list.map(elem => (
+          <a
+            key={elem.id}
+            href={`#user-${elem.id}`}
```

Comment:
Using href='#user-id' will cause the page to jump around when clicked. Consider using a button instead of an anchor tag if the intention isn't to navigate to another page or to a different part of the current page.